apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsArrayControl.tsx

```diff
@@ -275,27 +290,22 @@ export function JsonFormsArrayControl({
           path={composePaths(path, `${selectedIndex}`)}
           label={label}
           setSelectedIndex={setSelectedIndex}
+          isRemoveItemDisabled={minItems !== undefined && data <= minItems}
+          handleRemoveItem={handleRemoveItem(path, selectedIndex)}
```

Comment:
nit: might want to update `handleRemoveItem` to also have a isDisabled check -- users can easily remove the disabled prop on client and make the button clickable